library/std/convert.qs

```diff
@@ -264,4 +264,16 @@ namespace Microsoft.Quantum.Convert {
         );
     }
 
+    /// # Summary
```

Comment:
This change fails the CI check on Q# file formatting.  The error is "Run the formatter with the `--write` option to correct formatting for the above files." - can you give it a try? At a glance, sentences need to end with periods, same as in other API docs.

I would also add an `# Example` section, similar to that for `ResultArrayAsInt`, to clarify the use of the function.